js/script.js

```diff
@@ -0,0 +1,15 @@
+var triangle1Area = getTriangleArea(10, 15),
+	triangle2Area = getTriangleArea(3, 5),
+	triangle3Area = getTriangleArea(4, 2);
+
+function getTriangleArea(a, h) {
+	if (a>0 && h>0) {
+		return a*h/2;
+	} else {
+		console.log("Nieprawidłowe dane");
```

Comment:
else is unnecessary here - just write console.log ;) return in previous if statement stop function when true